static/app/views/issueList/addViewPage.tsx

```diff
@@ -45,8 +45,8 @@ const RECOMMENDED_SEARCHES: SearchSuggestion[] = [
 ];
 
 function AddViewPage({
-  personalSavedSearches,
-  organizationSavedSearches,
+  personalSavedSearches = [],
+  organizationSavedSearches = [],
 }: {
   organizationSavedSearches: SavedSearch[];
   personalSavedSearches: SavedSearch[];
```

Comment:
are the types wrong then

Reply:
looks like it https://github.com/getsentry/sentry/blob/master/static/app/views/issueList/overview.tsx#L1263-L1265